Source/WebCore/page/DOMWindow.idl

```diff
@@ -114,9 +114,8 @@
     [NotEnumerable, Conditional=IOS_GESTURE_EVENTS] attribute EventHandler ongestureend;
     [NotEnumerable, Conditional=IOS_GESTURE_EVENTS] attribute EventHandler ongesturestart;
 
-    // Non-standard: should probably deprecate this (https://bugs.webkit.org/show_bug.cgi?id=79653).
-    // Blink already deprecated it (https://bugs.chromium.org/p/chromium/issues/detail?id=437569).
-    CSSRuleList getMatchedCSSRules(optional Element? element = null, optional DOMString? pseudoElement = null);
+    // Non-standard: We only expose this to user worlds because Safari is using it.
```

Comment:
I think the comment should be more like:
`// Non-standard: For now we expose this to user worlds since Safari is using it, but we plan to remove it.`

Reply:
Fixed.